scene/gui/control.cpp

```diff
@@ -3558,9 +3550,12 @@ void Control::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "size_flags_stretch_ratio", PROPERTY_HINT_RANGE, "0,20,0.01,or_greater"), "set_stretch_ratio", "get_stretch_ratio");
 
 	ADD_GROUP("Localization", "");
-	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "auto_translate"), "set_auto_translate", "is_auto_translating");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "localize_numeral_system"), "set_localize_numeral_system", "is_localizing_numeral_system");
```

Comment:
Just food for thought - is this something that could be streamlined together with the added `auto_translate_mode`, maybe adding enum values that enable both translation + numbers localization?

May not be a good idea, but just leaving this here for consideration.

Reply:
As much as I like the idea, how many extra options that would add? I feel like it would make it a little convoluted.

Reply:
Yeah I don't really see how it would work to be honest :D

Reply:
Clamping it all together in a BitField of some sort sounds interesting...